website_sale_category_description/models/product_public_category.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ProductPublicCategory(models.Model):
+    _inherit = "product.public.category"
+
+    website_description_1 = fields.Html(
```

Comment:
Why not `website_description_top` (and `_bottom`)?

Reply:
I thought it but felt like not anchoring the field to a place would leave more room for theme designs, if some wanted to place these different.

But I'm not reluctant to name them like that if you think it's better.

Reply:
Well, their intention is clear, and designs can vary the exact place on top or on bottom, but to misuse them for putting for example both on top, or one in the middle, has no sense on the context of this module. If they do that, it can be still with the same fields. On the contrary, having `_1` and `_2`, you can have the doubt of which one represents the top part.